cider/exec/nextgen/jitlib/llvmjit/LLVMJITValue.h

```diff
@@ -32,41 +33,30 @@ class LLVMJITValue final : public JITValue {
 
  public:
   explicit LLVMJITValue(JITTypeTag type_tag,
-                        LLVMJITFunction& function,
+                        LLVMJITFunction& parent_function,
                         llvm::Value* value,
                         const std::string& name = "value",
                         JITBackendTag backend = JITBackendTag::LLVMJIT,
                         bool is_variable = false)
-      : JITValue(type_tag, name, backend)
-      , parent_function_(function)
+      : JITValue(type_tag, parent_function, name, backend)
+      , parent_function_(parent_function)
```

Comment:
you save `parent_function` in `LLVMJITValue` and `JITValue`, a bit redundant.

Reply:
Yes, but just a reference, not too bad.